lib/newrelic.js

```diff
@@ -67,32 +67,36 @@ class NewRelic {
      * @param {object} defaultMetrics default metrics to include in every New Relic event
      */
     constructor(options, defaultMetrics={}) {
+        const OPENWHISK_NEWRELIC_DISABLE_METRICS = process.env.OPENWHISK_NEWRELIC_DISABLE_METRICS || false;
+
         this.defaultMetrics = Object.assign(Metrics.openwhisk(), defaultMetrics);
 
         this.canSendMetrics = false;
-        if (!options ||
-            isBlankString(options.newRelicEventsURL) ||
-            isBlankString(options.newRelicApiKey)
-        ) {
-            console.error('Missing NewRelic events Api Key or URL. Metrics disabled.');
-
-        } else {
-            sendQueue.start(options.newRelicEventsURL, options.newRelicApiKey, options.sendIntervalMs);
-
-            // track this object per activation for global http metrics
-            if (activationVars && activationVars.active) {
-                activationVars.set(CLS_KEY_NEWRELIC, this);
-            }
 
-            if (options.actionTimeoutMetricsCb && typeof options.actionTimeoutMetricsCb !== 'function') {
-                console.error('Action timeout is not a proper function.');
-                delete options.actionTimeoutMetricsCb;
-            }
-            if (!options.disableActionTimeout && !process.env.DISABLE_ACTION_TIMEOUT_METRIC ) {
-                this.actionTimeoutHandlerId = sendMetricsOnActionTimeout(this, options.actionTimeoutMetricsCb);
-            }
+        if (OPENWHISK_NEWRELIC_DISABLE_METRICS !== true) {
```

Comment:
if(process.env.OPENWHISK_NEWRELIC_DISABLE_METRICS === undefined || process.env.OPENWHISK_NEWRELIC_DISABLE_METRICS  === null ||
process.env.OPENWHISK_NEWRELIC_DISABLE_METRICS === true){
  // disable
} else {
  // the whole metrics sending block from here
}